packages/api/database/migrations/1607463469723_message_schema.js

```diff
@@ -0,0 +1,33 @@
+/** @type {import('@adonisjs/lucid/src/Schema')} */
+const Schema = use('Schema');
+
+const { messageStatuses, messagesTypes } = require('../../app/Utils');
+
+class MessageSchema extends Schema {
+	up() {
+		this.create('messages', (table) => {
+			table.increments();
+			table
+				.integer('user_id')
+				.unsigned()
+				.references('id')
+				.inTable('users')
+				.onUpdate('CASCADE')
+				.onDelete('SET NULL');
```

Comment:
É mesmo este o comportamento esperado nesse situação? Não seria cascade? 🤔

Reply:
Seria no `onDelete('CASCADE')` ? Confess que ainda não entendi direito como isso funciona.

Reply:
Se um usuário for deletado, com o cascade, as mensagens pertencentes a ele também seriam. Com o set null, teríamos uma mensagem sem usuário, o que pode ser inútil ou até mesmo quebrar o front caso ele assuma que esse campo estará sempre presente.